pyproject.toml

```diff
@@ -34,7 +34,6 @@ exclude = ["htmlcov"]
 
 [tool.mypy]
 disallow_any_explicit = true
-disallow_any_generics = true
```

Comment:
Better to configure this just for the module in question rather than the whole project.